experiments/gaussian/gen.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
```

Comment:
this experiment was used to test out the CDF of the msdsl-based Gaussian noise generator.  i found that it matched well to +/- 6 sigma.  of course, there is more to random number generation than just matching the CDF, but this was already a big improvement as compared to the approach before using logarithmic compression